src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -26,12 +33,16 @@ public class UserAffiliationServiceImpl implements UserAffiliationService {
   private final UserTenantService userTenantService;
   private final TenantService tenantService;
   private final KafkaService kafkaService;
+  private final FolioExecutionContext folioExecutionContext;
+  private final FolioModuleMetadata folioModuleMetadata;
   private final ObjectMapper objectMapper = new ObjectMapper();
 
   @Override
   @SneakyThrows
   @Transactional
   public void createPrimaryUserAffiliation(String eventPayload) {
+    FolioExecutionContext requestContext = (FolioExecutionContext) folioExecutionContext.getInstance();
```

Comment:
Why is "getInstance" needed? Couldn't you just use folioExecutionContext itself?

Reply:
Before while saving user, it goes different context. It was need to keep initial context. Now It is removed based on new approach. 
![Uploading image.png…]()